apps/staging_sync/instructions.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+1. Merge your ETL branch to master and wait for the ETL to rebuild your dataset.
+2. Fill `Config` section and tick `Dry run` first to see what charts will be updated and hit `Sync charts`.
+3. Check the output, look for warnings that might indicate unexpected changes to charts in production.
+4. Untick `Dry run` and hit `Sync charts` again to apply changes to production.
+5. New charts were created as drafts, make sure to **publish them in production**.
```

Comment:
Again here, instead of "production", shouldn't it refer to whatever `Target` server the user specified?